src/schemaPostgres.ts

```diff
@@ -215,6 +218,10 @@ export class PostgresDatabase {
         'WHERE table_schema = $1 ' +
         'GROUP BY table_name ORDER BY lower(table_name)',
       [schemaName],
+      /**
+       * Customisations added:
```

Comment:
I'm not sure what this is referring to? Is there a change here?

Reply:
removing 👍 (was an old change I added but forgot to rm comment)